### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by vidhi-thakur**
- updated exports for components
- added authentication
- updated sidebar style

### CODE REVIEW

File: src/pages/login/Login.js [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+        setUserName(event.target.value)
+    }
+    const updatePassword = event => {
+        setPassword(event.target.value)
+    }
+    const submitLogin = (e) => {
+        e.preventDefault()
+        if (userName !== '' && password !== '') {
+            loginUser()
+            navigate(location?.state?.from?.pathname)
```
Comment (nyyshaaa) [inline src/pages/login/Login.js]: here use {replace:true} seprated by comma so that when navigating back through back button login doesn't show up again.

**Review by nyyshaaa — APPROVED** [review]
All else looks good to me. UI is cool.

File: src/pages/login/Login.js (hunk 1 above)
Comment (vidhi-thakur) [reply src/pages/login/Login.js]: thanks for your suggestion, updated the code

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Looks Good

**Pull request merged**